### ISSUE TITLE
LayoutLMv3 examples for CORD 

### ISSUE THREAD

**Issue by motive-ml** [microsoft/unilm#794]
Hi @HYPJUDY ~ 

I am interested in your research and thank you for providing the practical code.

I wanna verify LayoutLMv3 example code for CORD (f1 score : 0.9746, written in the paper)

but, when i run python script, The following error message was printed.

![image](https://user-images.githubusercontent.com/106065910/179183827-fd7d3205-9164-4b17-86fb-25bc61b7e7aa.png)

Can you provide train, test script for cord dataset and pretrained model ? 

Thanks.

**Comment by HYPJUDY** [comment]
Thank you for your interest!

According to the error message, something seemed wrong with loading the CORD dataset. Could you check if the dataset has been downloaded successfully? The code and comments below may give you some hints.
https://github.com/microsoft/unilm/blob/44273d47ac0971cd0ebe05335eb1e0043883b898/layoutlmv3/layoutlmft/data/cord.py#L51-L59

The code and scripts of CORD are almost the same as that of FUNSD. You can try it on FUNSD first to see if the problem exists. Then you can modify some training settings (e.g., dataset name, batch size, learning rate) for finetuning CORD.

What's more, @nielsrogge contributed a [model](https://huggingface.co/nielsr/layoutlmv3-finetuned-cord) and [renjithsasidharan](https://github.com/renjithsasidharan) contributed a [notebook](https://github.com/renjithsasidharan/layoutlmv3_cord/blob/main/Finetune_LayoutLMV3_on_CORD.ipynb) related to finetuning LayoutLMv3 on CORD with transformers.

**Comment by motive-ml** [comment]
In the case of the FUNSD dataset, the file is downloaded to the following path (`/home/user/.cache/huggingface/datasets/cord/cord/1.0.0/, *.arrow files`),

but in the case of CORD, the above error message occurs due to a URL problem.

Downloading the dataset directly to that path (`/home/user/.cache/huggingface/datasets/downloads/.../`) doesn't solve that problem.

renjithsasidharan's notebook is a good example, but the f1 score is too low. (f1 score : 0.92)

So, I need your code confirmation.

Thanks.

**Comment by Jyouhou** [comment]
I replaced `_get_drive_url("https://layoutlm.blob.core.windows.net/cord/CORD-1k-001.zip"),` with `"https://layoutlm.blob.core.windows.net/cord/CORD-1k-001.zip",`, following the comments, and it worked.
